src/AccessibilityInsights.SharedUx/Controls/EventConfigTabControl.xaml.cs

```diff
@@ -96,7 +96,7 @@ public void SetElement(A11yElement el)
                 RootNodes.Add(SuggestedNode);
             }
 
-            CustomNode = new EventConfigNodeViewModel("My Events");
+            CustomNode = new EventConfigNodeViewModel("My Events", isThreeState: true);
 
             CustomPropertiesNode = new EventConfigNodeViewModel("Properties") { Depth = 1 };
```

Comment:
I believe this node should be tri state as well, since it has children.
```suggestion
            CustomPropertiesNode = new EventConfigNodeViewModel("Properties", isThreeState: true) { Depth = 1 };
```

Reply:
Updated :)